### PR TITLE
Info on `mpi_force_c_code` feature

### PR DESCRIPTION
**Pull request by raoulstrackx**
Adds missing info on `mpi_force_c_code` feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ This is a list of the Cargo features available for mbedtls. Features in
                         will result in undefined instruction exceptions on
                         unsupported processors. On SGX, this feature is
                         enabled automatically.
+* *mpi_force_c_code* MbedTLS uses assembly code for MPI functions, when available.
```
Comment (jethrogb) [inline README.md:31]: This description belongs in the `mbedtls-sys` feature list. Here you can say that this enables the sys feature.

File: README.md [hunk 2]
```diff
@@ -95,6 +96,13 @@ This is a list of the Cargo features available for mbedtls-sys. Features in
 * *havege* Enable the Hardware Volatile Entropy Gathering and Expansion
            (HAVEGE) algorithm.
 * **legacy_protocols** Enable support for SSLv3, TLSv1.0 and TLSv1.1
+* *mpi_force_c_code* MbedTLS uses assembly code for MPI functions, when available.
+                     In some situations we may prefer C code instead. This is in
+                     particular the case on x86 platform where compile-time mitigation
```
Comment (Pagten) [inline README.md:101]: Nit: either "on the x86 platform" or "on x86 platforms"

**Review by Pagten — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -95,6 +96,13 @@ This is a list of the Cargo features available for mbedtls-sys. Features in
 * *havege* Enable the Hardware Volatile Entropy Gathering and Expansion
            (HAVEGE) algorithm.
 * **legacy_protocols** Enable support for SSLv3, TLSv1.0 and TLSv1.1
+* *mpi_force_c_code* MbedTLS uses assembly code for MPI functions, when available.
+                     In some situations we may prefer C code instead. This is in
+                     particular the case x86 platforms where compile-time mitigation
```
Comment (jethrogb) [inline README.md:101]: still typo here...

**Pull request merged**